app/assets/stylesheets/custom.css.scss

```diff
@@ -66,6 +66,10 @@ a:hover, a:active {
     color: $white;
 }
 
+#sign-in:hover, #sign-in:focus {
```

Comment:
Avoid using id selectors<br>Each selector in a comma sequence should be on its own line